app/controllers/easymon/checks_controller.rb

```diff
@@ -79,5 +80,9 @@ def show
       def add_prefix(result, message)
         result ? "OK #{message}" : "DOWN #{message}"
       end
+
+      def authorized
```

Comment:
The name `authorized` doesn't tell us what this method does. It reads like a declaration. How about `authorize`, or `authorize_request`?